src/ctags.d

```diff
@@ -28,101 +30,143 @@ void printCtags(File output, string[] fileNames)
 	foreach (fileName; fileNames)
 	{
 		File f = File(fileName);
-		if (f.size == 0) continue;
+		if (f.size == 0)
+			continue;
 		auto bytes = uninitializedArray!(ubyte[])(to!size_t(f.size));
 		f.rawRead(bytes);
 		auto tokens = getTokensForParser(bytes, config, &cache);
 		Module m = parseModule(tokens.array, fileName, null, &doNothing);
+
 		auto printer = new CTagsPrinter;
 		printer.fileName = fileName;
 		printer.visit(m);
 		tags ~= printer.tagLines;
 	}
-	output.write("!_TAG_FILE_FORMAT\t2\n"
-			~ "!_TAG_FILE_SORTED\t1\n"
-			~ "!_TAG_FILE_AUTHOR\tBrian Schott\n"
-			~ "!_TAG_PROGRAM_URL\thttps://github.com/Hackerpilot/Dscanner/\n");
+	output.write(
+		"!_TAG_FILE_FORMAT\t2\n" ~ "!_TAG_FILE_SORTED\t1\n" ~ "!_TAG_FILE_AUTHOR\tBrian Schott\n" ~ "!_TAG_PROGRAM_URL\thttps://github.com/Hackerpilot/Dscanner/\n");
 	tags.sort().copy(output.lockingTextWriter);
 }
 
 private:
 
-void doNothing(string, size_t, size_t, string, bool) {}
+/// States determining how an access modifier affects the parsning.
+/// Reset, when ascending the AST reset back to the previous access.
```

Comment:
If you put these lines inside the enum body, ddoc will associate the comments with the enum values.